Docker/entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
```

Comment:
`set -e` will be useful, so if one command will fail - the whole container will fail to start